test/tests/api/v2_0/nodes_tests.py

```diff
@@ -0,0 +1,370 @@
+    def test_node_workflows_get(self):
+        """ Testing GET:/api/2.0/nodes/:id/workflows """
+        resps = []
+        Api().nodes_get_all()
+        nodes = loads(self.__client.last_response.data)
+        for n in nodes:
+            if n.get('type') == 'compute':
+                Api().nodes_get_workflow_by_id(identifier=n.get('id'))
+                resps.append(self.__client.last_response.data)
+        for resp in resps:
+            assert_not_equal(0, len(loads(resp)), message='No Workflows found for Node')
+        assert_raises(rest.ApiException, Api().nodes_get_workflow_by_id, 'fooey')
+
+    @test(groups=['node_post_workflows-api2'], depends_on_groups=['node_workflows-api2'])
+    def test_node_workflows_post(self):
+        """ Testing POST:/api/2.0/nodes/:id/workflows """
+        resps = []
+        Api().nodes_get_all()
+        nodes = loads(self.__client.last_response.data)
+        for n in nodes:
+            if n.get('type') == 'compute':
+                Api().nodes_post_workflow_by_id(identifier=n.get('id'), name='Graph.Discovery',body={})
+                resps.append(self.__client.last_response.data)
+        for resp in resps:
+            assert_not_equal(0, len(loads(resp)), message='No Workflows found for Node')
+        assert_raises(rest.ApiException, Api().nodes_post_workflow_by_id, 'fooey',name='Graph.Discovery',body={})
+
+    @test(groups=['node_workflows_active-api2'], depends_on_groups=['node_post_workflows-api2'])
+    def test_node_workflows_active(self):
+        """ Testing GET:/api/2.0/nodes/:id/workflows/active """
+        resps = []
+        Api().nodes_get_all()
+        nodes = loads(self.__client.last_response.data)
+        for n in nodes:
+            if n.get('type') == 'compute':
+                Api().nodes_get_active_workflow_by_id(identifier=n.get('id'))
+                resps.append(self.__client.last_response.data)
+        for resp in resps:
+            assert_not_equal(0, len(loads(resp)), message='No active Workflows found for Node')
+        assert_raises(rest.ApiException, Api().nodes_get_active_workflow_by_id, 'fooey')
+
+    @test(groups=['node_workflows_del_active-api2'], depends_on_groups=['node_workflows_active-api2'])
+    def test_node_workflows_del_active(self):
+        """ Testing DELETE:/api/2.0/nodes/:id/workflows/active """
+        Api().nodes_get_all()
+        nodes = loads(self.__client.last_response.data)
+        for n in nodes:
+            if n.get('type') == 'compute':
+                id = n.get('id')
+                assert_is_not_none(id)
+                Api().nodes_del_active_workflow_by_id(identifier=id)
+                assert_equal(0, len(self.__client.last_response.data),
+                        message='No active Workflows found for Node {0}'.format(id))
+        assert_raises(rest.ApiException, Api().nodes_del_active_workflow_by_id, 'fooey')
+
+    @test(groups=['node_tags_patch'])#, depends_on_groups=['nodes.api2.discovery.test'])
```

Comment:
You probably want a nodes test dependency.   Just to squelch related error noise caused by dependent failures.